Remotion/ObjectBinding/Web.UnitTests/UI/Controls/Validation/BocListValidationResultDispatchingValidatorTest.cs

```diff
@@ -160,6 +161,169 @@ public void DispatchValidationFailures_WithRowEditModeControlHavingInvalidValida
+                typeof(BusinessObjectBoundEditableWebControlValidationResultDispatchingValidator)
+              }));
+
+      ((BaseValidator)editableRowValidators[0]).Validate();
+
+      NamingContainer.Controls.Add(bocList);
+      NamingContainer.Controls.Add(validator);
+
+      var validationResultMock = new Mock<IBusinessObjectValidationResult>();
+      validationResultMock
+          .Setup(e => e.GetValidationFailures(bocList.DataSource.BusinessObject, bocList.Property, true))
+          .Returns(Array.Empty<BusinessObjectValidationFailure>())
+          .Verifiable();
+
+      validationResultMock
+          .Setup(_ => _.GetValidationFailures((IBusinessObject)values[0], simpleColumnDefinition.GetPropertyPath().Properties.Single(), true))
+          .Returns(Array.Empty<BusinessObjectValidationFailure>())
+          .Verifiable();
+      validationResultMock
+          .Setup(_ => _.GetUnhandledValidationFailures(It.IsAny<IBusinessObject>(), It.IsAny<bool>(), It.IsAny<bool>()))
+          .Returns(Array.Empty<BusinessObjectValidationFailure>());
+
+      validator.DispatchValidationFailures(validationResultMock.Object);
+
+      var bocListValidationFailureRepository = bocList.ValidationFailureRepository;
+      var editedRow = ((IBocList)bocList).EditModeController.GetEditableRow(0);
+
+      var failuresInRepository = bocListValidationFailureRepository.GetUnhandledValidationFailuresForBocListAndContainingDataRowsAndDataCells(true);
+      Assert.That(failuresInRepository.Count, Is.EqualTo(1));
+      Assert.That(failuresInRepository.Single().ColumnDefinition, Is.EqualTo(simpleColumnDefinition));
+      Assert.That(failuresInRepository.Single().RowObject, Is.EqualTo(values[0]));
+      Assert.That(failuresInRepository.Single().Failure.ErrorMessage, Is.EqualTo("The value contains unsupported characters: \"\u0001\" (position: 1)."));
+
+      validationResultMock.Verify();
+
+      // Validates that dispatch was called to dispatching validator
+      var bocTextValue = ((Control)editableRow).FindControl("0") as BusinessObjectBoundEditableWebControl;
+      validationResultMock.Verify(_ => _.GetValidationFailures(bocTextValue.DataSource.BusinessObject, bocTextValue.Property, true));
+    }
+
+    [Test]
+    public void DispatchValidationFailures_WithoutErrorIndicatorColumn_DoesNotDispatchToRepository ()
+    {
+      IBusinessObject businessObject = TypeWithReference.Create();
+      var businessObjectProperty = businessObject.BusinessObjectClass.GetPropertyDefinition("ReferenceList");
+
+      var bocList = new BocListMock();
+      bocList.ID = "someBocList";
+      bocList.DataSource = new StubDataSource(businessObject.BusinessObjectClass);
+      bocList.DataSource.BusinessObject = businessObject;
+      bocList.Property = (IBusinessObjectReferenceProperty)businessObjectProperty;
+
+      var validationResultMock = new Mock<IBusinessObjectValidationResult>(MockBehavior.Strict);
```

Comment:
Nothing wrong with the test, but I think we should discuss as a team how we want to handle such "no calls" mock objects. I think it might be good to have a special name and also to add a `.VerifyNoOtherCalls()` at the end of the test. this makes it clearer that we assert that nothing is called on this mock. Otherwise, this fact is only visible when investigating the test in detail and realizing that this strict mock with no setups is here to ensure nothing was called.

Reply:
Yeah, I agree with having the verify no other calls. 
As for names, do you suggest having something like a stub/fake/etc. postfix in such cases? 

In that case I have 3 suggestions:
* requisite 
* dependency
* extra

I think extra would be the most appropriate as it suggests that it is not needed in this case.

@a-ctor, @MichaelKetting thoughts?

Reply:
Added a discussion item for our next weekly meeting :)

Reply:
Okay, I see there's "many" calls. With "many" calls, adding an excpetion for each would be more involved. If it's just the three methods I can see in the next test, it's probably a cointoss. if it's five or more, it's no fun to use the exception trick.